deepnote_toolkit/sql/sql_execution.py

```
@@ -263,13 +264,28 @@ class ExecuteSqlError(Exception):
+    retries = Retry(
+        total=3,
+        backoff_factor=0.5,
+        status_forcelist=[500, 502, 503, 504],
+        allowed_methods=["POST"],
+    )
+    session.mount("http://", HTTPAdapter(max_retries=retries))
+    session.mount("https://", HTTPAdapter(max_retries=retries))
+    return session
+
+
 def _generate_temporary_credentials(integration_id):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_generate_temporary_credentials`

(ANN202)